cmd/terramate/cli/cli.go

```diff
@@ -783,15 +834,23 @@ func (c *cli) createStack() {
 	log.Info().Msgf("created stack %s", stackPath)
 	c.output.Msg(out.V, "Created stack %s", stackPath)
 
-	report := generate.Do(c.cfg(), stackDir)
+	report, vendorReport := c.gencodeWithVendor(stackDir)
 
```

Comment:
you're not calling `vendorReport.RemoveIgnoredByKind()`. Is there a reason?

Reply:
In this case we only output the reports if tehre is any errors on them, when there is no errors they go to output as -v, so I thought that for -v it was OK to keep everything since it is verbose output. But maybe on the HasFailures case it would also make sense to remove the ignored, was just not that sure here :thinking: